elide-contrib/elide-dynamic-config-helpers/src/main/resources/elideDBConfigSchema.json

```diff
@@ -128,7 +128,7 @@
                     "url": "jdbc:mysql://localhost/elide?serverTimezone=UTC",
                     "driver": "com.mysql.jdbc.Driver",
                     "user": "guest1",
-                    "dialect": "org.hibernate.dialect.MySQL5Dialect"
+                    "dialect": "com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.impl.HiveDialect"
```

Comment:
I can see why you switched to enums earlier.  "com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.impl.HiveDialect" is a beast to type in.

Maybe we can have the best of both worlds - an enum for dialects we support - and the ability to override with a custom class.  I'm fine if we want to make that work a separate PR.

Reply:
updated to have both